libpod/diff.go

```diff
@@ -29,6 +29,13 @@ func (r *Runtime) getLayerID(id string) (string, error) {
 	var toLayer string
 	toImage, err := r.GetImage(id)
 	if err != nil {
+		if id == "**latest" {
+			ctr, err := r.LookupContainer(id)
```

Comment:
Instead of lookupContainer, can you use GetLatestContainer() instead here?

Reply:
We also really need to port this to use r.LookupContainer instead of r.store.Container, can you add a TODO for that?

Reply:
ok